Resources/Prototypes/Reagents/Consumable/Drink/alcohol.yml

```diff
@@ -906,6 +906,22 @@
         reagent: Ethanol
         amount: 0.15
 
+- type: reagent
+  id: Eggnog
+  name: reagent-name-eggnog
+  parent: BaseAlcohol
+  desc: reagent-desc-eggnog
+  physicalDesc: reagent-physical-desc-fizzy-and-creamy
+  flavor: eggnog
+  color: "#e6d6bc"
+  metamorphicSprite:
+    sprite: Objects/Consumable/Drinks/eggnogglass.rsi
+    state: icon_empty
+  metamorphicMaxFillLevels: 4
+  metamorphicFillBaseName: fill-
+  metamorphicChangeColor: true
+  fizziness: 0.25
```

Comment:
```suggestion
```
Is eggnog fizzy? I though it is not sparkling, but thick and creamy. Same for the physical description.